src/api/app/assets/javascripts/webui/request.js

```diff
@@ -134,11 +134,10 @@ $(document).ready(function(){
 
 function loadDiffs(element){
   $('.loading-diff').removeClass('invisible');
-  var id = element.data('action-id');
-  var number = element.data('req-number');
+  var url = element.data('path');
   var index = element.data('index');
   var diffLimit = $('.sourcediff').data('diff-limit');
-  var url = '/request/'+ number + '/request_action/' + id + '?index=' + index;
+  url = url + '?index=' + index;
```

Comment:
We can combine the lines 137 and 140.

```suggestion
var url = element.data('url') + '?index=' + index;
```

Reply:
Done.